logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -253,6 +254,7 @@ internal class ConversationDataSource internal constructor(
                         )
                     },
                     selfTeamIdProvider().getOrNull(),
+                    lastNotificationDate = DateTimeUtil.currentIsoDateTimeString()
```

Comment:
Maybe we shouldn't set it here at all, because we use that `persistConversations` when syncing, right? So during every sync we will update this `lastNotificationDate` to the current device time and won't get notified about messages that happened before that time, or maybe I'm missing something 🤔